<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TableWidget</class>
 <widget class="QTableWidget" name="TableWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1044</width>
    <height>704</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TableWidget</string>
  </property>
  <widget class="QWidget" name="centralwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1041</width>
     <height>561</height>
    </rect>
   </property>
   <widget class="QTableWidget" name="employeeTable">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>1001</width>
      <height>431</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>First Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Last Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>City</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>State</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Zip Code</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>SSN</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Withholdings</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Salary</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>401</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:28pt; font-weight:600; color:#55aa00;&quot;&gt;Employee Status&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="returnEmpl">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>50</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Return</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1044</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuRoster">
    <property name="title">
     <string>Roster</string>
    </property>
   </widget>
   <addaction name="menuRoster"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>3</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="addEmployee">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>600</y>
     <width>351</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Employee</string>
   </property>
  </widget>
  <widget class="QPushButton" name="remEmployee">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>600</y>
     <width>351</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove Employee</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
